lib/makeTwemojiRenderer.js

```diff
@@ -60,4 +53,29 @@ function merge() {
 		}
    }
    return out
+}
+
+function getSource(options, icon){
+	var source = ''
+
+	if (options.baseUrl.indexOf('http') !== 0) {
+		source += normalizeProtocol(options.protocol)
+	}
+
+	var baseUrl = options.baseUrl.endsWith('/') ? options.baseUrl : options.baseUrl + '/'
```

Comment:
We shouldn't use endsWith as it would break with older browsers

Reply:
https://github.com/appfigures/react-easy-emoji/pull/26/commits/480e477ead4cde8b3226d5bb1f3c8b497f3440a6